CMakeLists.txt

```diff
@@ -229,7 +229,11 @@ ENDIF(WANT_TAP)
 
 # check for CARLA
 IF(WANT_CARLA)
-	PKG_CHECK_MODULES(CARLA carla-standalone>=1.9.5)
+	PKG_CHECK_MODULES(CARLA carla-native-plugin)
+	# look for carla under old name
+	IF(NOT CARLA_FOUND)
+		PKG_CHECK_MODULES(CARLA carla-standalone>=1.9.5)
```

Comment:
if we really keep this, even though I am not totally okay with it, please bump it to 1.9.11
it will then target the soon-to-be-stable version.
at some later point this version should be changed to 2.0.0, when carla is ready.